suite/tests/CMakeLists.txt

```diff
@@ -2166,7 +2166,11 @@ if (AARCH64 AND UNIX AND ZLIB_FOUND AND NOT ANDROID) # TODO i#1874: Add drcaches
     # Test TLB simulator with a virtual to physical v2p.textproto mapping.
     set(srcdir
       ${PROJECT_SOURCE_DIR}/clients/drcachesim/tests/drmemtrace.threadsig.aarch64.raw)
-    set(locdir ${PROJECT_BINARY_DIR}/drmemtrace.threadsig.aarch64)
+    set(locdir ${CMAKE_CURRENT_BINARY_DIR}/drmemtrace.tlb_simulator_v2p.aarch64)
+    # We are fine with a static config-time creation of the directory and the
+    # first test run creating the trace/ files with subsequent test runs re-using
+    # those files, as this test targets the tlb simulator.  If our srcdir had
+    # trace/ file we would just use those.
     file(REMOVE_RECURSE ${locdir})
     file(MAKE_DIRECTORY ${locdir})
     file(COPY ${srcdir}/raw DESTINATION ${locdir}/)
```

Comment:
Oh, I thought every test had similar steps: set local dir, delete it, mkdir, copy files.
I didn't realize some tests do need clobbering.
Good catch, thanks!

Reply:
Seemed like a race from the failure output so did not take long to find the other test using the same dir name.